client_delivery/shared/src/commonMain/kotlin/presentation/map/MapScreenModel.kt

```diff
@@ -0,0 +1,47 @@
+package presentation.map
```

Comment:
Don't forget to add todo comments for remove delay functions

Reply:
if all the remaining code is ok, plz give us your approval ! 
else : plz review the code and give us your opinion
@mosayed01